TodoList/src/app/screens/ToDoList/index.js

```diff
@@ -1,63 +1,64 @@
-import React, { Component } from 'react'
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import { View, ScrollView, StyleSheet, TextInput, Text } from 'react-native'
-import { connect } from 'react-redux'
-import { Actions } from 'react-native-router-flux'
+import { View } from 'react-native';
+import { connect } from 'react-redux';
 
-import { actionCreators } from '../../../redux/actions'
-import Title from '../Title'
-import Footer from '../Footer'
-import List from '../List'
-import Input from '../Input'
-import { styles } from './styles.js'
+import { actionCreators } from '../../../redux/actions';
+import Title from '../../components/Title';
+import Footer from '../../components/Footer';
+import List from '../../components/List';
+import Input from '../../components/Input';
+import { styles } from './styles';
 
 class ToDoApp extends Component {
-    addItem = (item) => {
-    const {dispatch} = this.props
-    dispatch(actionCreators.addItem(item))
-  }
-  removeItem = (index) => {
-    const {dispatch} = this.props
-    dispatch(actionCreators.removeItem(index))
-  }
-  toggleItem = (index) => {
-    const {dispatch} = this.props
-    dispatch(actionCreators.toggleItem(index))
-  }
-  removeCompleted = () => {
-    const {dispatch} = this.props
-    dispatch(actionCreators.removeCompleted())
-  }
-  
-render() {
-  const {items} = this.props
-  return (
-    <View style={styles.container}>
-      <Title> Todo List </Title>
+  render() {
+    const { items } = this.props;
+    const { addItem } = this.props;
+    const { removeItem } = this.props;
+    const { toggleItem } = this.props;
+    return (
+      <View style={styles.container}>
+        <Title> Todo List </Title>
         <Input
-          placeholder={'Enter an item!'}
-          onSubmit={this.addItem}
+          placeholder="Enter an item!"
+          onSubmit={addItem}
         />
-        <View style={styles.divider}/>
         <List
           items={items}
-          onRemoveItem={this.removeItem}
-          onToggleItemCompleted={this.toggleItem}
+          onRemoveItem={removeItem}
+          onToggleItemCompleted={toggleItem}
         />
         <View style={styles.divider} />
```

Comment:
You don't need this `View`. If you want a divider, you can just add that style to the adjacent components